packages/widget/src/components/substrate-account-selector/styles.ts

```diff
@@ -0,0 +1,81 @@
+import { css } from 'lit';
+
+export const substrateAccountSelectorStyles = css`
+  // Custom option for dropdown. We are using part as we pass the custom option
+  // to the dropdown component as a property
+  dropdown-component::part(dropdown) {
+    border-radius: 2.5rem;
+  }
+
+  dropdown-component::part(dropdownWrapper) {
+    padding: 0;
+  }
+
+  dropdown-component::part(customOptionContent) {
+    display: flex;
+    flex-direction: column;
+  }
+
+  dropdown-component::part(customOptionContentName) {
+    margin-bottom: 0.25rem;
+    font-size: 0.875rem;
+    font-weight: 500;
+  }
+
+  dropdown-component::part(customOptionContentMain) {
+    display: flex;
+    align-items: center;
+  }
+
+  dropdown-component::part(identicon) {
+    max-width: 3rem;
+    width: fit-content;
+  }
+
+  dropdown-component::part(customOptionContentAccountData) {
+    display: flex;
+    flex-direction: column;
+    margin-left: 0.5rem;
+  }
+
+  dropdown-component::part(customOptionContentType) {
+    margin-bottom: 0.16rem;
+    font-size: 0.75rem;
+    font-weight: 500;
+  }
+
+  dropdown-component::part(customOptionContentAddress) {
+    color: #334155;
```

Comment:
color variable por favor

Reply:
sin problema

Reply:
it is so funy to see some spanish words here 🤣